stern/tail.go

```diff
@@ -264,6 +265,53 @@ func (t *Tail) Print(msg string) {
 	fmt.Fprint(t.out, buf.String())
 }
 
+func (t *Tail) GetResumeRequest() *ResumeRequest {
+	if t.last.timestamp == "" {
+		return nil
+	}
+	return &ResumeRequest{Timestamp: t.last.timestamp, LinesToSkip: t.last.lines}
+}
+
+func (t *Tail) consumeLine(line string) {
+	rfc3339Nano, content, err := splitLogLine(line)
+	if err != nil {
+		t.Print(fmt.Sprintf("[%v] %s", err, line))
+		return
+	}
+
+	// PodLogOptions.SinceTime is RFC3339, not RFC3339Nano.
+	// We convert it to RFC3339 to skip the lines seen during this timestamp when resuming.
+	rfc3339 := removeSubsecond(rfc3339Nano)
+	t.rememberLastTimestamp(rfc3339)
+	if t.resumeRequest.shouldSkip(rfc3339) {
+		return
+	}
+
+	if t.Options.IsExclude(content) || !t.Options.IsInclude(content) {
+		return
+	}
+
+	if t.Options.Timestamps {
+		updatedTs, err := t.Options.UpdateTimezone(rfc3339Nano)
+		if err != nil {
+			t.Print(fmt.Sprintf("[%v] %s", err, line))
+			return
+		}
+		t.Print(updatedTs + " " + content)
+		return
+	}
+	t.Print(content)
```

Comment:
I prefer to do the following, but I'll leave it up to you.

```suggestion
	msg = content
	if t.Options.Timestamps {
		updatedTs, err := t.Options.UpdateTimezone(rfc3339Nano)
		if err != nil {
			t.Print(fmt.Sprintf("[%v] %s", err, line))
			return
		}
		msg = updatedTs + " " + msg
	}
	t.Print(msg)
```

Reply:
I prefer it too. Thanks